.h5pignore

```diff
@@ -3,8 +3,9 @@ src
 .github
 .gitignore
 .h5pignore
+.eslintignore
```

Comment:
The `.eslintignore` file is not required at all given your scripts in `package.json`, so the entry here is not necessary either.